benchmark-overhead/src/test/java/io/opentelemetry/results/PrintStreamPersister.java

```diff
@@ -4,66 +4,64 @@
     out.println("----------------------------------------------------------");
     out.println(" Run at " + new Date());
     out.printf(" %s : %s\n", config.getName(), config.getDescription());
-    out.printf(
-        " %d users, %d iterations\n",
-        config.getConcurrentConnections(), config.getTotalIterations());
+    out.printf(" %d users, %d iterations\n", config.getConcurrentConnections(), config.getTotalIterations());
     out.println("----------------------------------------------------------");
 
-    display(sorted, "Agent", appPerfResults -> appPerfResults.agent.getName());
-    display(sorted, "Avg. CPU (user)", res -> String.valueOf(res.averageJvmUserCpu));
-    display(sorted, "Max. CPU (user)", res -> String.valueOf(res.maxJvmUserCpu));
-    display(sorted, "Startup time (ms)", res -> String.valueOf(res.startupDurationMs));
-    display(sorted, "Total allocated MB", res -> format(res.getTotalAllocatedMB()));
-    display(sorted, "Heap (min)", res -> String.valueOf(res.heapUsed.min));
-    display(sorted, "Heap (max)", res -> String.valueOf(res.heapUsed.max));
-    display(sorted, "Thread switch rate", res -> String.valueOf(res.maxThreadContextSwitchRate));
-    display(sorted, "GC time", res -> String.valueOf(res.totalGCTime));
-    display(sorted, "Req. mean", res -> format(res.requestAvg));
-    display(sorted, "Req. p95", res -> format(res.requestP95));
-    display(sorted, "Iter. mean", res -> format(res.iterationAvg));
-    display(sorted, "Iter. p95", res -> format(res.iterationP95));
-    display(sorted, "Net read avg (bps)", res -> format(res.averageNetworkRead));
-    display(sorted, "Net write avg (bps)", res -> format(res.averageNetworkWrite));
-    display(sorted, "Peak threads", res -> String.valueOf(res.peakThreadCount));
+    display(results, "Agent", appPerfResults -> appPerfResults.agent.getName());
+    display(results, "Run duration", res -> {
+      Duration duration = Duration.ofMillis(res.runDurationMs);
+      return String.format("%02d:%02d:%02d", duration.toHours(), duration.toMinutesPart(), duration.toSecondsPart());
+    });
+    display(results, "Avg. CPU (user)", res -> String.valueOf(res.averageJvmUserCpu));
+    display(results, "Max. CPU (user)", res -> String.valueOf(res.maxJvmUserCpu));
+    display(results, "Avg. mch tot cpu", res -> String.valueOf(res.averageMachineCpuTotal));
+    display(results, "Startup time (ms)", res -> String.valueOf(res.startupDurationMs));
+    display(results, "Total allocated MB", res -> format(res.getTotalAllocatedMB()));
+    display(results, "Heap (min)", res -> String.valueOf(res.heapUsed.min));
+    display(results, "Heap (max)", res -> String.valueOf(res.heapUsed.max));
+    display(results, "Thread switch rate", res -> String.valueOf(res.maxThreadContextSwitchRate));
+    display(results, "GC time", res -> String.valueOf(res.totalGCTime));
+    display(results, "GC pause time", res -> String.valueOf(NANOSECONDS.toMillis(res.totalGcPauseNanos)));
```

Comment:
Can we add unit to those metrics that don't have them (and are not totally obvious)?
```suggestion
    display(results, "GC time (ms)", res -> String.valueOf(res.totalGCTime));
    display(results, "GC pause time (ms)", res -> String.valueOf(NANOSECONDS.toMillis(res.totalGcPauseNanos)));
```

Reply:
Sure. It's in the main table, but it's probably helpful to add it here so I'll do that. 👍🏻